garrysmod/lua/includes/modules/halo.lua

```diff
@@ -59,15 +60,25 @@ function Render( entry )
 			render.SetStencilPassOperation( STENCIL_REPLACE )
 			render.SetStencilFailOperation( STENCIL_KEEP )
 			render.SetStencilZFailOperation( STENCIL_KEEP )
-
+			
+			if v.SuppressLighting then
+				render.SuppressEngineLighting(true)
+			end
 				for k, v in pairs( entry.Ents ) do
-
+			
 					if ( !IsValid( v ) ) then continue end
-
+			
+					RenderEnt = v
+			
 					v:DrawModel()
-
+			
 				end
-
+				
+					RenderEnt = NULL

```

Comment:
Your repo is based off an old commit while the code in the file is from a newer one.